src/pages/Settings/PaymentSettings.vue

```diff
@@ -0,0 +1,339 @@
+<template>
+	<www-page class="payments" :gray-background="true">
+		<the-my-kiva-secondary-menu slot="secondary" />
+		<div class="title-area">
+			<div class="row column">
+				<h1>
+					Payment Methods
+				</h1>
+			</div>
+		</div>
+		<div class="row">
+			<kv-settings-card class="columns small-12 large-8" title="Credit Card Settings">
+				<template v-slot:icon>
+					<kv-icon
+						class="icon"
+						title="Payment"
+						name="unknown-card"
+					/>
+				</template>
+				<template v-slot:content>
+					<form class="payment-settings-default-form" @submit.prevent>
+						<fieldset>
+							<legend v-if="hasSavedPaymentMethods">
+								Select your default card
+							</legend>
+							<legend v-else>
+								There are no cards saved to this account
+							</legend>
+							<div v-for="(paymentMethod, index) in savedPaymentMethods"
+								:key="index"
+								class="payment-settings-default-form__cc-wrapper"
+							>
+								<kv-radio
+									class="payment-settings-default-form__radio"
+									:id="`creditCard-${index}`"
+									:radio-value="paymentMethod.nonce"
+									v-model="selectedDefaultCardNonce"
+									:checked="selectedDefaultCardNonce === paymentMethod.nonce"
+								>
+									<img class="payment-settings-default-form__cc-icon"
+										:src="paymentMethod.imageUrl"
```

Comment:
Suggest `alt=""` since paymentMethod.description is screen-readable

Reply:
Ohh woops, I see your screenshot now. I'd recommend setting alt="{{ Visa, MasterCard, Kohls Cash, etc. here}}" to describe the image instead.

However, from a UX perspective, I think using text instead of the card logo is more direct and clear to the the user. eg
* MasterCard ending in 1234   vs
* (image of MasterCard logo) ending in 1234

Perhaps best is 
* (image of MasterCard logo) MasterCard ending in 1234

Reply:
There are some technical limitations here, we actually don't know what type of card it is, just the image URL (provided by braintree)  that is why we never actually say "Visa" "MasterCard" etc

Reply:
Oh, bummer OK. Probably alt="" then to prevent the screenreader from reading the filename of the image.

Reply:
> Oh, bummer OK. Probably alt="" then to prevent the screenreader from reading the filename of the image.

How about `alt="Credit Card Logo"` ?

Reply:
Nice, maybe just "credit card" though. Thinking of how it would read, "Image, Credit Card" 
"ending in 1234" sounds pretty good.